crates/web-client/test/address.test.ts

```diff
@@ -0,0 +1,144 @@
+import { expect, Page } from "@playwright/test";
+import test from "./playwright.global.setup";
+import { AddressInterface, AccountId, Address, NetworkId } from "../js";
+const instanceAddress = async ({
+  page,
+  accountId,
+  _interface,
+}: {
+  page: typeof Page;
+  accountId?: typeof AccountId;
+  _interface: typeof AddressInterface;
+}) => {
+  return await page.evaluate(
+    async ({ accountId, _interface }) => {
+      let _accountId;
+      const client = window.client;
+      if (accountId) {
+        _accountId = accountId;
+      } else {
+        const newAccount = await client.newWallet(
+          window.AccountStorageMode.private(),
+          true
+        );
+        _accountId = newAccount.id();
+      }
+      const address = window.Address.fromAccountId(_accountId, _interface);
+      return address.interface();
+    },
+    { accountId, _interface }
+  );
+};
+
+const instanceAddressTestBech32 = async (page: Page, bech32Prefix: string) => {
```

Comment:
nit: I would rename this function to better differentiate from `instanceAddressFromBech32`, also the prefix should be `networkId`.

```suggestion
const instanceNewAddressBech32 = async (page: Page, networkId: string) => {
```